ci-templates/gitlab/image-builder.yml

```diff
@@ -34,7 +64,7 @@ default:
 
 .docker: &docker
   - docker info
-  - docker login -u $DOCKER_REGISTRY_USER -p $DOCKER_REGISTRY_PASSWORD $DOCKER_REGISTRY
+  - cat ../plain.docker.json | jq -r ".password" | docker login -u $(cat ../plain.docker.json | jq -r ".username") --password-stdin $DOCKER_REGISTRY
```

Comment:
The line is too long, use a multi-line string here.

Reply:
When I switched to using the values without removing the `_unencrypted`, I changed this to a multi-line string